starwars/src/App.js

```diff
@@ -52,7 +52,7 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        <h1 className="Header">REACT <br/> WARS</h1>
+        <h1 className="Header">REACT WARS</h1>
```

Comment:
the style look cool, but it not readable 
![screencapture-r489q9l9oo-codesandbox-io-2019-02-16-05_01_31](https://user-images.githubusercontent.com/37554037/52884358-b441f900-31a8-11e9-8655-814ca1b7a23e.png)